app/views/admin/settings/index.html.erb

```diff
@@ -1,103 +1,97 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('Archive Settings') %></h2>
+<h2 class="heading"><%= t(".heading") %></h2>
 <!--/descriptions-->
 
 <!--main content-->
-<%= form_for @admin_setting, :url => admin_setting_path(@admin_setting), html: { method: "put", class: "manage" } do |f| %>
+<%= form_for @admin_setting, html: { class: "manage" } do |f| %>
   <%= error_messages_for @admin_setting %>
+  <fieldset>
+    <legend><%= t(".landmark.account_and_invitations") %></legend>
```

Comment:
Could we use `legend` instead of `landmark` in the key? 

Landmark headings are only necessary on forms where the legend is hidden -- screen readers won't read the legend text, but they _will_ read the heading. (At least, this was the case 10 years ago; it may have changed.) Our [rule](https://github.com/otwcode/otwarchive/blob/156720eb90c391d183f598edf531f940b0fd5fe3/public/stylesheets/site/2.0/07-interactions.css#L554) is that if there are more than 3 field sets on a form, the `verbose` class should be added to the form, which will make the legend visible (and available to screen readers). 

This form has more than 3 field sets, so we will one day add the `verbose` class to the form and yanking out the unnecessary landmark headings. When that day comes, it makes more sense to have the key use `legend` in its name than the name of the thing we removed.